Automattic/pocket-casts-android
# Adds a clear Up Next confirmation when long pressing on the Mini Player

This change did the following
- Improve the clear Up Next confirmation to match iOS
- Improve the Mini Player long press popup to match iOS
- Add a clear Up Next confirmation to the Mini Player

<img  width="300" src="https://user-images.githubusercontent.com/308331/184861937-5dd5cc41-e302-41ed-838b-a20fc646b084.png" />
<img  width="300" src="https://user-images.githubusercontent.com/308331/184861950-676b2684-a0de-41af-b340-f3961ebcce04.png" />
<img  width="300" src="https://user-images.githubusercontent.com/308331/184861961-91177322-404b-42be-87b9-dc86b0db54e2.png" />

Fixes https://github.com/Automattic/pocket-casts-android/issues/240
Fixes https://github.com/Automattic/pocket-casts-android/issues/237

https://user-images.githubusercontent.com/308331/184861792-cfbe78a7-473d-4fe4-8127-b0804b6bef5c.mov

**Test Steps**
1. Add more than two episodes to your Up Next
2. Long press the Mini Player
3. Tap "Close and clear Up Next"

The clear Up Next confirmation should be displayed.
